src/main/java/eu/dissco/backend/service/MachineAnnotationServiceService.java

```diff
@@ -103,6 +108,21 @@ public JsonApiListResponseWrapper scheduleMass(JsonNode flattenObjectData, List<
         new JsonApiMeta(scheduledMasRecords.size()));
   }
 
+  private void validateBatchingRequest(Boolean batchingRequested,
+      List<MachineAnnotationServiceRecord> masRecords)
+      throws ConflictException {
+    if (Boolean.FALSE.equals(batchingRequested)) {
```

Comment:
This could have been `!batchingRequested` right? Why did you choose for this way?

Reply:
Sonar recommends doing it this way to avoid NPEs